### DISCUSSION TITLE
Clean Architecture without SPA

### DISCUSSION THREAD

**Discussion by EhsanHabib** [jasontaylordev/CleanArchitecture#666] [Ideas]
Hey @jasontaylordev, wouldn't it be nice to create a separate solution template for only Web Api because not everyone in the world is using Angular as a Client Application.

**Comment by MedBouc** [comment]
I second this or better put **clientApp** in its own separate project
As the trend is clearly BLAZOR, tomorrow it will be easy to set up a blazor project using the API

**Comment by jasontaylordev** [comment]
The template now supports three options:

1. Angular SPA with Web API 
2. React SPA with Web API
3. Web API only

Create a new solution from the command-line:

```bash
dotnet new ca-sln
```

Options:

```bash
-n, --name        The name of the output being created (default is output directory)
-o, --output        Location to place the generated content
-cf, --client-framework        The type of client framework to use
-us, --use-sqlite        Whether to use SQLite instead of LocalDB
```

Example

```bash
dotnet new ca-sln -cf React
```
